src/lib/Scenes/Home/Components/ForYou/index.tsx

```diff
@@ -86,8 +82,11 @@ export class ForYou extends React.Component<ViewProperties & RelayProps, State>
     }
   }
 
-  registerRailModule = (railModule: RailModule) => {
-    this.railModules.push(railModule)
+  /**
+   * Can be used to register a mounting module or de-register an unmounting module by setting the index to `null`.
+   */
+  registerRailModule(index: number, railModule: RailModule) {
+    this.railModules[index] = railModule
```

Comment:
Allow the rail to be removed from the list of mounted rails by passing `null` as `railModule`.